modules/infinite-scroll/infinity.php

```diff
@@ -829,7 +829,7 @@ function action_wp_footer_settings() {
 
 			// Check if the taxonomy is attached to one post type only and use its plural name.
 			// If not, use "Posts" without confusing the users.
-			if ( count( $taxonomy->object_type ) < 2 ) {
+			if ( $taxonomy && count( $taxonomy->object_type ) < 2 ) {
```

Comment:
This may not be enough to fix this, since `WP_Taxonomy` has a `null` `$object_type` by default:
https://core.trac.wordpress.org/browser/tags/5.4/src/wp-includes/class-wp-taxonomy.php#L149

What would you think about checking for `WP_Taxonomy`, and then making sure we have an array and not just `null`?

```php
/*
 * Check if the taxonomy is attached to one post type only and use its plural name.
 * If not, use "Posts" without confusing the users.
 */
if (
	is_a( $taxonomy, 'WP_Taxonomy' )
	&& is_countable( $taxonomy->object_type )
	&& count( $taxonomy->object_type ) < 2
) {
	$post_type = $taxonomy->object_type[0];
}
```